sphinx/themes/basic/static/_compat.js

```diff
@@ -0,0 +1,133 @@
+/*
+ * _compat.js
+ * ~~~~~~~~~~
+ *
+ * Compatability shim for jQuery and underscores.js.
+ *
+ * WILL BE REMOVED IN Sphinx 6.0
```

Comment:
nit: Mention `RemovedInSphinx60Warning` in this comment, since it is likely what someone would search for when determining what all to remove as part of preparing for Sphinx 6.0.